### PR TITLE
Add layout

### PR DESCRIPTION
**Pull request by gvonkoss**
This removes a conditional operation involving keys to display documentation — not sure what the reasoning was, but is related to: https://github.com/Financial-Times/origami-image-service/pull/235/files

Removing it as it is documentation and not an active endpoint, pls discuss

**before**
image-service-before

**after**
image-service-after

### CODE REVIEW

**Comment by notlee** [comment]
o-layout ❤️

File: public/main.css [hunk 1]
```diff
@@ -1,32 +1,44 @@
...
+	padding: 0;
 }
 
-/* Header overrides */
-@media (min-width: 46.25em) {
-	.o-header__primary__left {
-		width: 303px;
-	}
+.o-forms__additional-info {
+	padding-bottom: 0.5em;
```
Comment (notlee) [inline public/main.css]: Is this needed? Hoping `o-forms` defaults are sensible 🙏

Comment (notlee) [reply public/main.css]: Same with `o-forms__radio+.o-forms__label`

Comment (gvonkoss) [reply public/main.css]: I'm not sure why but
**with custom css**
screen shot 2018-10-05 at 10 52 09

**without custom css**
screen shot 2018-10-05 at 10 52 24

Comment (gvonkoss) [reply public/main.css]: unless I'm  using the wrong class for this 🤔

Comment (gvonkoss) [reply public/main.css]: hmmmmmmmmmmm this markup is way outta date by the looks of things

**Pull request merged**